For review (#5)
lsst-ops/LPM-251: checklist.tex

\newlist{todolist}{itemize}{2}
\setlist[todolist]{label=$\square$}

> \section{Checklist for iDAC}\label{sec:checklist}

Reviewer: ```suggestion
% \section{IDAC Proposal Checklist}\label{sec:checklist}
% Section heading is in main source file LPM-251.tex
```

> There are a spectrum of possibilities for a DAC from Just hosting Object lite to full copy.

Reviewer: ```suggestion
There is a spectrum of possibilities for an IDAC's scope, from just hosting the Object lite database, to serving full copies of the current data release and prompt products database.
```

> Here an attempt is made to have a set of check lists  that can be used to look at an iDAC.

Reviewer: ```suggestion
In this section we provide a set of checklists, each of which can be used in the initial design of an IDAC --- at the proposal stage, for example.
```

> \secref{sec:fullDAC} covers a full up iDAC while  \secref{sec:liteDAC} is the minimum.

Reviewer: ```suggestion
\secref{sec:fullDAC} covers a full capability iDAC, while  \secref{sec:liteDAC} gives the criteria for a minimum capability IDAC.
```

Author: ok

> There will undoubtedly be proposals in between.

Reviewer: ```suggestion
There will undoubtedly be proposals in between. There are some criteria that any IDAC must meet, in order to comply with Rubin Observatory data policy. These are given in \secref{anyDAC}.
```

\subsection{Any DAC} \label{sec:anyDAC}
\begin{todolist}
\item Authentication/Authorization system  inline with Rubin Observatory Access
\item Agreement to make broadly accessible to all Data Rights holders
\end{todolist}

\subsection{Lite iDAC} \label{sec:liteDAC}

> Items in \secref{sec:anyDAC} as well as :

Reviewer: ```suggestion
All criteria in \secref{sec:anyDAC}, and then, in addition:
```

\begin{todolist}
\item Database system capable of handling $4^{10}$ rows.
\item IVOA TAP interface, MyDB and Table Upload, CAOM support.
\item About 500TB of disk for catalogs + MyDBs.
\item Professional support staff (min 0.25 FTE)
\item Sufficient connectivity to support users
\end{todolist}

\subsection{Full DAC} \label{sec:fullDAC}

> Items in \secref{sec:anyDAC} as well as :

Reviewer: ```suggestion
All criteria in \secref{sec:anyDAC}, and then, in addition:
```

\begin{todolist}
\item Staff (about 5 FTE) to handle major hardware installation
\item Agreement to stand up standard Science Platform (Puppet/Kubernetes etc.)
\item Database system capable of handling all catalogs (or Qserv) with IVOA  interfaces.
\item Understanding of sizing model in \citeds{DMTN-135} - sizing model and a cost model for the iDAC.

> \item Sufficient funding for 10 years of operations  based on cost model and FTE (probably $>\$6M/year$).

Reviewer: Better to stipulate sufficient funding to cover the LSST operations period, FY22-FY34, I think? This is a hard ask: we may need to soften the wording a little. No-one gets funded for 10 years up front.

Reviewer: ```suggestion
\item "Commitment to fund the IDAC through the LSST operations period, FY22-FY34 (probably  $>\$6M/year$ based on hardware cost model and labor plan).".
```
